test/SILOptimizer/peephole_thick_to_objc_metatype.sil

```diff
@@ -115,3 +117,47 @@ bb0(%0 : $T):
   strong_release %0 : $T
   return %3 : $@thick T.Type
 }
+
+// CHECK-LABEL: sil @$test_peephole_objc_to_thick_to_objc
+// CHECK: [[T:%.*]] = apply
+// CHECK-NOT: {{objc_to_thick_metatype }}
```

Comment:
Do you really need to use a regex here? CHECK-NOT can take bare strings without a regex.